tests/integration_tests/data_steward/metrics/required_labs_test.py

```diff
@@ -189,39 +188,3 @@ def test_get_lab_concept_summary_query(self):
                          msg='Compare the number '
                          'of labs submitted '
                          'in the measurement')
-
```

Comment:
I'm not sure how this relates to the submission directory naming convention tests.

Reply:
This test was failing when I pushed my changes, and since Chao and I had discussed merging it earlier, I thought I could do it here. However, we now have #421 which does this, and once it is merged, the commits pertaining to this will not be necessary. I'll remove them and push